pkg/sentry/platform/kvm/bluepill_unsafe.go

```diff
@@ -57,170 +55,3 @@ func uintptrValue(addr *byte) uintptr {
 func bluepillArchContext(context unsafe.Pointer) *arch.SignalContext64 {
 	return &((*arch.UContext64)(context).MContext)
 }
-
-// bluepillHandler is called from the signal stub.
-//
-// The world may be stopped while this is executing, and it executes on the
-// signal stack. It should only execute raw system calls and functions that are
-// explicitly marked go:nosplit.
-//
-//go:nosplit
-func bluepillHandler(context unsafe.Pointer) {
```

Comment:
why do we need to move this in arch-specific files? In this case, we duplicate this code, maybe we can avoid this.

Reply:
Currently, there are at least 2 features that are arch-specific: mmio_exit, irq_injection.
I will try to keep the logic of the code consistent with qemu and find a good way to ensure that there is no duplicated code.
Thanks.